server/src/server.ts

```
@@ -8,8 +8,37 @@
 import config from './config/index.js';
+import { withFileLock } from './utils/fileLock.js';
+import { registerTrainingBundleRoute } from './routes/trainingBundleRoute.js';
 
-const __dirname = path.dirname(fileURLToPath(import.meta.url));
+function resolveServerModuleDir(): string {
+  if (typeof __dirname !== 'undefined') {
+    return __dirname;
+  }
+  let metaUrl: string | undefined;
+  try {
+    metaUrl = (0, eval)('import.meta.url') as string;
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)